index.html

```diff
@@ -1,11 +1,61 @@
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CALYX</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="style.css">
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
+        <div class="nav-brand">
+          <div class="h3 text-bold color-primary box-3d p-1-x bg-secondary">
+            CALYX Mart
+          </div>
+        </div>
+        <div class="nav-content">
+          <!-- <h3>Navbar</h3>
+                  <p>Text</p>
+                  <div class="submenu">
+                    <input id="submenu-input" type="checkbox" />
+                    <label for="submenu-input" class="submenu-text">
+                      Submenu
+                    </label>
+                    <ul>
+                      <li>Option 1</li>
+                      <li>Option 12</li>
+                      <li>Option 123</li>
+                    </ul>
+                  </div> -->
```

Comment:
please remove this unused lines of comments

Reply:
will do